hot-reload.md

```diff
@@ -23,13 +23,12 @@ physical or virtual device can be the target.
 1.  Modify one of the Dart files in your project. Most types of code changes can
 be hot reloaded; for a list of changes that require a full restart, see
 [Understanding limitations](#understanding-limitations).
-1.  If you're working in an IntelliJ IDE that supports Flutter's IDE tools,
-either click the Hot Reload button on the toolbar:
+1.  If you're working in an IntelliJ IDE that supports Flutter's IDE tools, select Save All (`cmd-s`/`ctrl-s`), or
+click the Hot Reload button on the toolbar:
 
-![alt_text](/images/intellij/hot-reload.gif "image_tooltip")
+   ![alt_text](/images/intellij/hot-reload.gif "image_tooltip")
 
-or select **File > Save All**. If you're running the app at the command line 
-using `flutter run`, type `r` in the terminal window. 
+   If you're running the app at the command line using `flutter run`, type `r` in the terminal window. 
```

Comment:
Missing bold formatting for File > Save All. https://developers.google.com/style/ui-elements

Reply:
Done